src/content/effective-dart/style.md

```diff
@@ -454,8 +454,8 @@ formatting the code by hand and trying to make it more readable. Think of
 `dart format` as a partnership where you work together, sometimes iteratively, 
 to produce beautiful code.
 
-
-### AVOID lines longer than 80 characters
+<a id="avoid-lines-longer-than-80-characters"></a>
+### PREFER lines 80 characters or less
```

Comment:
Grammar nit: "or fewer". :)

Reply:
Phew, good thing I'm not a technical writer or something 😅